src/Extractor.php

```diff
@@ -62,7 +42,7 @@ class Extractor
     private Crawler $crawler;
 
     private Document $document;
-    private OEmbed $oembed;
+    protected OEmbed $oembed;
```

Comment:
Because `Embed\Adapters\Instagram\Extractor` and `Embed\Adapters\Facebook\Extractor` are setting this property.
I can replace it with a setter if you want to.

Reply:
Ah, ok. It's fine, I didn't change this in my commit and didn't get any error, I don't know why.